README.md

```diff
@@ -19,3 +19,11 @@ Then run the tests with:
 ```bash
 make integration-test
 ```
+
+# Build Multi-Architecture Docker Images with Buildx
+
+Please refer to https://docs.docker.com/buildx/working-with-buildx/ to enable Buildx.
+
+Run `make builder` to create Buildx instance before starting to build the images.
+
+Run `make images` to start build the images.
```

Comment:
Thanks for adding this section. Can we provide more context around why and when a user will need to run `make builder`? How about something like:

In some local environments like Ubuntu, where the default Buildx builder uses the `docker` driver, the `make images` command might fail with the following error:

```
$ make images
multiple platforms feature is currently not supported for docker driver. Please switch to a different driver (eg. "docker buildx create --use")
Makefile:57: recipe for target 'images' failed
make: *** [images] Error 1
```

To fix this, you can create a new Buildx builder instance by running `make builder`. This command will create a builder that uses the `docker-container` driver that can build multi-platform images. For more information, see the Buildx builder [documentation](https://docs.docker.com/buildx/working-with-buildx/#work-with-builder-instances).